powerscale/provider/synciq_peer_certificate_resource_test.go

```diff
@@ -83,10 +83,19 @@ func TestAccSyncIQPeerCertificateResource(t *testing.T) {
 					path = "/ifs/invalid.pem"
 				}
 				`,
+				PreConfig: func() {
+					if FunctionMocker != nil {
+						FunctionMocker.Release()
```

Comment:
I am not sure how `Release()` works I usually just use unpatch().

But if that works as well all good !

Reply:
I am also not sure, lemme use unpatch.

Reply:
done